contracts/vaults/RibbonThetaVault.sol

```diff
@@ -390,8 +390,15 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
             expiry = getNextFriday(IOtoken(oldOption).expiryTimestamp());
         }
 
+        IStrikeSelection strikeSelection = IStrikeSelection(strikeSelection);
+
         (uint256 strikePrice, uint256 delta) =
-            IStrikeSelection(strikeSelection).getStrikePrice(expiry, isPut);
+            lastStrikeOverride == round
+                ? (overridenStrikePrice, strikeSelection.delta())
```

Comment:
Should not have round = 0 because then for first v2 round will use overridden. we should start with round = 1